Fix for NetBox supperuser account creation and small LDAP and mail cleanup

drybjed opened the pull request.

drybjed committed:
[core] Include domain in admin private e-mails

drybjed committed:
[saslauthd] Use correct syntax for realm variable

This commit fixes a bug introduced in[1] which was meant to enable
authentication using e-mail address syntax (user@domain). It
unfortunately broke authentication of mail relays via LDAP directory.
The use of '%d' instad of '%r' might have been influenced by use of '%d'
in Postfix 'ldap_table' syntax.

[1]: https://github.com/debops/debops/commit/6e4b85a9069d727edf45eaa87af99c3983aca45d

drybjed committed:
[ldap] Don't create unused objects in 'ldap' role

drybjed committed:
[LDAP] Update structural objectClass in roles

drybjed committed:
[netbox] Make 'git clone' task more resilient

drybjed committed:
[netbox] Fix creation of the superuser account

ypid reviewed (dismissed):
The NetBox fix looks good to me. Thanks!

drybjed committed:
[icinga_web] Fix role execution without MariaDB

On hosts without MariaDB, the Icinga Web installation will omit x509
plugin support since it only works with MariaDB. This patch makes sure
that Ansible has sane defaults for 'ansible_local.mariadb.*' variables
which are not present in such case.

drybjed committed:
[librenms] Fix application install procedure

This patch updates the LibreNMS installation procedure to make sure that
the application installs correctly.

drybjed committed:
[ci] Disable 'no_log' keywords in test environment

Ansible tasks with no_log enabled are cumbersome to debug in the hard to
access CI pipeline. They are especially tricky to fix when the issue
cannot be replicated during development. With this patch, roles that use
the 'debops__no_log' global variable will show everything during
testing.

drybjed committed:
[opendkim] Use 'debops__no_log' variable
